app/scrapers/bing.py

```diff
@@ -30,3 +34,39 @@ def parse_response(self, soup):
         print('Bing parsed: ' + str(urls))
 
         return urls
+
+    def parse_video_response(self, soup):
```

Comment:
This method can be changed to a static method by using `@staticmethod` and removing `self` from the position argument.

Reply:
Since previously it was not useful so not using this time too.

Reply:
That is a really poor answer.  The suggested modification is indeed useful as the linter is trying to help you to avoid unnecessary runtime overhead.

```
Normal: 0.5956953540444374
 Class: 0.38477163994684815
Static: 0.18970419897232205  # Three times faster than the normal method call.
```
```python
#!/usr/bin/env python3

from timeit import timeit


setup = """
class TestClass:
    def add_normal_method(self, i):
        return i + 1

    @classmethod
    def add_class_method(cls, i):
        return i + 1

    @staticmethod
    def add_static_method(i):
        return i + 1


t = TestClass()"""


print('Normal:', timeit('t.add_normal_method(1)', setup))
print(' Class:', timeit('t.add_class_method(1)', setup))
print('Static:', timeit('t.add_static_method(1)', setup))
```